### DISCUSSION TITLE
Add support for staking rewards

### DISCUSSION THREAD

**Discussion by lammetak** [ghostfolio/ghostfolio#1274] [Ideas]
I see you are adding dividends to Ghostfol.io but I miss the option to staking rewards. That's really awesome :).

It would be awesome if you could add staking rewards automatically. But for now manual option to add staking rewards could do it for me.

**Comment by VibeNL** [comment]
I would love to see staking rewards. Currently trying to add them as dividends but not having much luck.

**Comment by ukd1** [comment]
I also need this to be even able to use this

**Comment by brhahlen** [comment]
Adding my upvote for this as well.

**Comment by Xosten** [comment]
Upvoting this as well

**Comment by luluwebmaster** [comment]
I'm bringing this topic up a bit, it would be a really practical feature !

**Comment by nik110403** [comment]
is this still happening?
